Wireframe/css/style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: 100px;
+  margin-right: 100px;
+  margin-bottom: 0;
+  background: #f0efe7;
+  padding: 0;
+  height: 100%;
+}
+
+header{
+  text-align: center;
+  margin-bottom: 5px;
+ 
+}
+h1,h2{
+  margin:5px;
+ 
+}
+article{
+  padding-left: 20px;
+}
+
+article:first-child {
+  margin-bottom: 10px;
+  border: 1.0px solid white;
+  border-radius: 15px;
+  height: auto;
+}
+
+#article-one-h2-margin{
+  margin-left: 80px;
+
+}
+#article-first-child-title{
+  display: flex;
+}
+
+#other-article-title{
+  display: flex;
+  width: 991px;
+  margin:5px;
+  border: 0.1px solid white;
```

Comment:
it's better to use whole numbers here instead of decimals. `0.1px` wide border would not be visible well.